lib/gds_api/mapit.rb

```diff
@@ -37,6 +37,10 @@ def areas
     def postcode
       @response["postcode"]
     end
+
+    def country_name
+      @response["country_name"]
+    end
```

Comment:
Is this meant to be in local_links_manager.rb not mapit.rb?

Reply:
Thanks Sean - interesting question.  No, I don't believe so.  An instance of `Location` is created with the response from the `/postcode/` API call in `location_for_postcode` within this file.  That is [called in frontend](https://github.com/alphagov/frontend/blob/7572a3ac88d7f86969bb9ef48ed0d53e0af2e148/app/controllers/local_transaction_controller.rb#L62) in the `LocalTransactionController` which is the controller we need to update in order to surface the 'unavailable in country' error.

Reply:
From chatting in person - we've recommended creating two separate commits, one for the test stubs for local links manager and the another for the Mapit response. Remove reference about adding country name to local links manager adapter - as these endpoint return the whole JSON blob anyway.